charts/gitea-org-runner/templates/deployment.yaml

```
@@ -1,22 +1,24 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "gitea-runner-ttd.fullname" . }}
+  name: {{ include "gitea-org-runner.fullname" . }}
   labels:
-    {{- include "gitea-runner-ttd.labels" . | nindent 4 }}
+    {{- include "gitea-org-runner.labels" . | nindent 4 }}
 spec:
+  {{- if not .Values.autoscaling.enabled }}
   replicas: {{ .Values.replicaCount }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)